packages/truffle-core/lib/commands/help.js

```diff
@@ -1,6 +1,6 @@
 var command = {
   command: "help",
-  description: "Display a list of all commands or information about a specific command",
+  description: "Display a list of all commands or info about a specific command",
```

Comment:
why shorten this to "info"? "information" may be more understandable to people for whom English is not their first language since info is slang.

Reply:
I changed this to make it fit on one line.  When `truffle help` is used, there previously were too many characters in the string and it wrapped to the next line.  We could try to shorten it some other way...

Reply:
Maybe `List all commands or provide information about a specific command`